control-services/bin/setup.sh

```diff
@@ -27,9 +27,6 @@ sudo rm -rf /usr/local/go
 wget https://go.dev/dl/go1.17.7.linux-amd64.tar.gz -P /tmp/
 sudo tar -C /usr/local/ -xzf /tmp/go1.17.7.linux-amd64.tar.gz
 
-# Check go version
-go version
```

Comment:
Move this to the bottom after you modify the path.  Its a good debugging tool/status check to have in setup

Reply:
Moved.